storage/sas_token_test.go

```diff
@@ -0,0 +1,98 @@
+package storage
+
+import "testing"
+
+func TestParseStorageAccountConnectionString(t *testing.T) {
+	testCases := []struct {
+		input               string
+		expectedAccountName string
+		expectedAccountKey  string
+	}{
+		{
+			"DefaultEndpointsProtocol=https;AccountName=azurermtestsa0;AccountKey=2vJrjEyL4re2nxCEg590wJUUC7PiqqrDHjAN5RU304FNUQieiEwS2bfp83O0v28iSfWjvYhkGmjYQAdd9x+6nw==;EndpointSuffix=core.windows.net",
+			"azurermtestsa0",
+			"2vJrjEyL4re2nxCEg590wJUUC7PiqqrDHjAN5RU304FNUQieiEwS2bfp83O0v28iSfWjvYhkGmjYQAdd9x+6nw==",
+		},
+	}
+
+	for _, test := range testCases {
```

Comment:
Similar case of `for` with `_`

Reply:
(same here) this'd be int, value so we can't do this here afaik